src/__mocks__/api-handlers.js

```diff
@@ -104,4 +104,13 @@ export const handlers = [
         }),
     )
   }),
+
+  rest.post('https://api.github.com/repos/:org/:repo/issues', (req, res, ctx) => {
+    // if (req.params.org !== 'pablo-mayrgundter' || req.params.repo !== 'Share' || req.params.issueNumber !== '17') {
```

Comment:
remove?

Reply:
will remove it 
Attempted to write those tests did not succeed.